test/integration/puppetlabs/services/jruby/puppet_environments_int_test.clj

```diff
@@ -83,7 +83,7 @@
 
 (defn catalog-contains?
   [catalog resource-type resource-title]
-  (let [resources (get-in catalog ["data" "resources"])]
+  (let [resources (get catalog "resources")]
```

Comment:
Ah, I meant to separate that into it's own commit. What's being returned is no longer

```
{ ...
  "data": { "resources": [...], ... }
}
```
it's now just `{ ... "resources": [...], ... }`

This seems to have been changed as part of removing the PSON document type: https://github.com/puppetlabs/puppet/commit/7007b6f96ef953dffbf14851b37fdbd388fdc2d7